specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1189 @@
+          "in" : "query",
+          "description" : "(optional) if set to true, response will contain input and document level statistics.",
+          "required" : false,
+          "type" : "boolean"
+        }, {
+          "in" : "body",
+          "name" : "languageBatchInput",
+          "description" : "Collection of documents to analyze.",
+          "required" : false,
+          "schema" : {
+            "$ref" : "#/definitions/LanguageBatchInput"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "A successful call results in the detected language with the highest probability for each valid document",
+            "examples" : {
+              "application/json" : {
+                "documents" : [ {
+                  "id" : "1",
+                  "detectedLanguages" : [ {
+                    "name" : "English",
+                    "iso6391Name" : "en",
+                    "score" : 1
+                  } ]
+                }, {
+                  "id" : "2",
+                  "detectedLanguages" : [ {
+                    "name" : "French",
+                    "iso6391Name" : "fr",
+                    "score" : 1
+                  } ]
+                }, {
+                  "id" : "3",
+                  "detectedLanguages" : [ {
+                    "name" : "Spanish",
+                    "iso6391Name" : "es",
+                    "score" : 1
+                  } ]
+                }, {
+                  "id" : "4",
+                  "detectedLanguages" : [ {
+                    "name" : "(Unknown)",
+                    "iso6391Name" : "(Unknown)",
+                    "score" : 0
+                  } ]
+                } ],
+                "errors" : [ ]
+              }
+            },
+            "schema" : {
+              "$ref" : "#/definitions/LanguageBatchResult"
+            }
+          },
+          "500" : {
+            "description" : "Error Response",
+            "schema" : {
+              "$ref" : "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/sentiment" : {
+      "post" : {
+        "summary" : "Sentiment",
+        "description" : "The API returns a document sentiment prediction, as well as sentiment scores for each sentiment class (Positive, Negative, and Neutral).In addition, sentence level sentiment predictions are returned. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/text-analytics-supported-languages\">Supported languages in Text Analytics API</a> for the list of enabled languages.",
+        "operationId" : "56f30ceeeda5650db055a3c9",
+        "consumes" : [ "application/json", "text/json" ],
+        "produces" : [ "application/json", "text/json" ],
+        "parameters" : [ {
+          "name" : "modelVersion",
+          "in" : "query",
+          "description" : "(optional) This value indicates which model be used for scoring .",
```

Comment:
we may want to work on making this a little more descriptive or add alink to see where the versions are where that is ready